Apps/APN/ios/MyAppPushNotificationsHandler.swift

```diff
@@ -1,25 +1,22 @@
 import Foundation
 import CioMessagingPushAPN
-import UserNotifications
 import CioTracking
 
 @objc
 public class MyAppPushNotificationsHandler : NSObject {
 
   public override init() {}
 
-  @objc(setupCustomerIOClickHandling:)
-  public func setupCustomerIOClickHandling(withNotificationDelegate notificationDelegate: UNUserNotificationCenterDelegate) {
+  @objc(setupCustomerIOClickHandling)
```

Comment:
Changed this file to what RN customers will begin using when the iOS feature launches.